modules/programs/qutebrowser.nix

```
@@ -0,0 +1,61 @@
+{ config, lib, pkgs, ... }:
+let
+  cfg = config.programs.qutebrowser;
+
+  formatLine = o: n: v:
+    let
+      formatValue = v:
+        if builtins.isNull v then
+          "None"
+        else if builtins.isBool v then
+          (if v then "True" else "False")
+        else if builtins.isString v then
+          ''"${v}"''
+        else
+          builtins.toString v;
+    in if builtins.isAttrs v then
+      lib.concatStringsSep "\n" (lib.mapAttrsToList (formatLine "${o}${n}.") v)
+    else
+      "${o}${n} = ${formatValue v}";
+in {
+  options.programs.qutebrowser = {
+    enable = lib.mkEnableOption "programs.qutebrowser";
+
+    settings = lib.mkOption {
+      type = lib.types.attrs;
+      default = { };
+      description = ''
+        Options to add to Qutebrowser <filename>config.py</filename> file.
+        See <link xlink:href="qute://help/settings.html">qute://help/settings.html</link> for options.
```

I would suggest using HTTP links here instead to allow people not using qute at the moment to also read the documentation. For example
```suggestion
        See <link xlink:href="https://qutebrowser.org/doc/help/settings.html"/> for options.
```

---

I didn't know setting are available there :) Done.